src/AppBundle/Controller/ProjectController.php

```diff
@@ -25,10 +26,58 @@ public function typeaheadAction(Request $request)
 
         $q = $request->query->get('q', '');
 
-        return new JsonResponse(array_values(array_map(function ($project) {
-            return $project['name'];
+        $response = new JsonResponse(array_values(array_map(function ($project) {
+            return $project;
         }, array_filter($this->get('projects')->getAllProjects(), function ($project) use ($q, $siteNames) {
-            return (empty($q) || stripos($project['name'], $q) !== false) && !in_array($project['name'], $siteNames) ;
+            return (empty($q) || stripos($project['name'], $q) !== false);
+            //            return (empty($q) || stripos($project['name'], $q) !== false) && !in_array($project['name'], $siteNames) ;
         }))));
+        $response->headers->addCacheControlDirective('must-revalidate', true);
+        return $response;
+    }
+
+    /**
+     * @Route("/project/members/", name="project_members")
+     * Method("POST")
+     */
+    public function membersAction(Request $request)
+    {
+        if ($request->get('project')) {
```

Comment:
is that a query parameter? 

1. be specific use `$request->query->get()`
2. Maybe put it in the url `/project/{id}/members`?